src/Providers/DocumentCreation/Smbc/SmbcTextfileDocumentCreator.cs

```diff
@@ -12,5 +12,15 @@ public byte[] CreateDocument(List<string> fileContent)
         {
             throw new System.NotImplementedException();
         }
+
+        public byte[] CreateHtmlDocument(List<string> fileContent)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public byte[] CreatePdfDocument(List<string> fileContent)
+        {
+            throw new System.NotImplementedException();
+        }
```

Comment:
Is this file actually implementing anything? Is it used? Cut it out if not needed.